hack/cluster-down.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+#
+# This file is part of the Kepler project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2022 The Kepler Contributors
+#
+./cluster-up/cluster-down.sh
```

Comment:
why we keep this file?

Reply:
This is because to make the Makefile only execute the scripts in `hack/` folder....
Then, the those scripts can execute scripts in other folders....

It is not 100% necessary, but a good way to structure the project....

Reply:
ok